_config.yml

```diff
@@ -2,9 +2,10 @@ markdown: kramdown
 sass:
   style: :compressed
 exclude:
-- README.md
-- wp-content/README.md
-- vendor
+  - README.md
+  - wp-content/README.md
+  - vendor
+  - node_modules
```

Comment:
This doesn't seem right... why not a shrinkwrap or yarn.lock file?

Reply:
ime this makes builds take forever

Reply:
Actually, this file didn't need to be committed since there's no Node stuff going on here. The `node_modules` exclusion came from a little experiment that I later deleted—except for this file. I will revert the `_config.yml` file to its previous state to remove confusion.